docs/sphinx/_toc.yml.in

```diff
@@ -0,0 +1,72 @@
+# Anywhere {branch} is used, the branch name will be substituted.
+# These comments will also be removed.
+defaults:
+  numbered: False
+  maxdepth: 6
+root: index
+subtrees:
+  - entries:
+    - file: what-is-omnitrace.rst
+
+  - caption: API
+    entries:
+    - file: doxygen/html/files
+      title: API library
+    - file: doxygen/html/globals
+      title: Functions
+    - file: doxygen/html/annotated
+      title: Data structures
```

Comment:
The convention across other ROCm component docs is to put **Install** and **How to** at the top. Would be nice to reorder the sidebar for consistency.

Reply:
Some of the components have Tutorials and How-to after install, so I'll only move the API links down and place them beside reference.